ToolBox/KaikoDataConverter/Program.cs

```diff
@@ -306,17 +336,18 @@ private static IEnumerable<Tick> ParseKaikoTradeFile(Symbol symbol, string unzip
         /// Parse the quantity field of the kaiko ticks - can sometimes be expressed in scientific notation
         /// </summary>
         /// <param name="lineParts">The line from the Kaiko file</param>
-        /// <param name="quantityColumn">The index of the quantity column </param>
+        /// <param name="column">The index of the quantity column </param>
         /// <returns>The quantity as a decimal</returns>
-        private static decimal ParseQuantity(string[] lineParts, int quantityColumn)
+        private static decimal ParseScientificNotationToDecimal(string[] lineParts, int column)
         {
-            var quantity = lineParts[quantityColumn];
-            if (quantity.Contains("e"))
+            var value = lineParts[column];
+            if (value.Contains("e"))
             {
-                return Decimal.Parse(quantity, System.Globalization.NumberStyles.Float);
+                return Decimal.Parse(value, System.Globalization.NumberStyles.Float);
```

Comment:
I would add the `CultureInfo.InvariantCulture` argument on this line for non-US users